scripts/dockerbot/__init__.py

```diff
@@ -3,6 +3,10 @@
 
 
 def parse_pr_metadata(pr_body):
+    if not pr_body:
+        pr_body = ''
+    if not isinstance(pr_body, basestring):
+        print "pr_body '%s' is not a string, %s" % (pr_body, type(pr_body))
```

Comment:
Should we just return here, there is a logger, maybe we should log the error and return {}